index.js

```diff
@@ -1,6 +1,8 @@
 //https://github.com/angular/angular.js/pull/10732
 
-var angular = require('angular');
+if (typeof angular !== 'undefined') {
```

Comment:
Are you really aware what `require()` do?)
It doesn't execute anything, just returns an entity(function in our case)